### DISCUSSION TITLE
my dataset is VOC07 format.  but my dataset has only one class,what should i do ?

### DISCUSSION THREAD

**Discussion by henbucuoshanghai** [facebookresearch/detectron2#2511] [Q&A]
where to change the detection chasses to my own class?

and only one class is OK?

**Comment by jdegeus** [comment]
You should define your classes in your dataset. One class is no problem. How to load pascal voc is answered here: #991

**Comment by henbucuoshanghai** [comment]
when a image has no object in it ,i use it as the background.i still put it in the train.txt&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;

and it has error :
FileNotFoundError: [Errno 2] No such file or directory: 'datasets/VOC2007/Annotations/234.xml'






when a image has no instances, I have to remove it from the traintxt?????
help !!!&nbsp; &nbsp; &nbsp; thank you!!
